docs/usage-wharfyml/step-types/docker.md

```diff
@@ -7,13 +7,14 @@ docker:
   file: src/api/Dockerfile # path to Dockerfile from root of repo
   tag: latest,${GIT_COMMIT},${GIT_TAG}
   # Optional arguments
-  name: # defaults to step name
-  group: default
+  destination: # defaults to "${registry}/${group}/${REPO_NAME}/${name}", or "${registry}/${group}/${REPO_NAME}" if ${REPO_NAME}==${name}
+  name: # defaults to the step name
+  group: # defaults to ${REPO_GROUP} (the Wharf group name)
   context: src/api # context to use when building (defaults to root of repo)
   secret: gitlab-registry # kubernetes secret to use when pushing image
-  registry: https://harbor.local # base url of registry
+  registry: harbor.local # base url of registry, defaults to ${REG_URL}
```

Comment:
Just so I understand, this isn't a change in behavior but the example of default value was wrong, meaning `${REG_URL}` never had the protocol in it?

Reply:
That's right. Having the protocol there actually results in an invalid image name. Forgot to mention it in the description